src/components/heading/RoomsFilter.js

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import {useContext} from 'react';
+import {Context} from "../../Context";
+import MySelectRoom from "./MySelectRoom";
+
+
+
+function RoomsFilter() {
+
+    const {currentRoomQuant, setCurrentRoomQuant} = useContext(Context);
+
+    const setRo = (currentRoomQuant) => {
+  
+        console.log(currentRoomQuant);
```

Comment:
Don't leave console.logs when you're out from local development, abstract client should not be able see such stuff